packages/devtools_app/lib/src/app.dart

```diff
@@ -516,6 +518,55 @@ class SettingsDialog extends StatelessWidget {
             toggle: preferences.toggleVmDeveloperMode,
             gaItem: gac.vmDeveloperMode,
           ),
+          Column(
+            children: [
+              Row(
+                children: [
```

Comment:
we may want to add an indicator to let the user know that they have verbose logging on, and that it may affect devtools performance.
Maybe we need to discuss turning it off after a certain amount of time?

Reply:
Added an issue for this as a follow up:
https://github.com/flutter/devtools/issues/5435